extension-style-app/src/androidTest/java/com/mapbox/maps/testapp/style/layers/generated/CircleLayerTest.kt

```diff
@@ -706,6 +706,72 @@ class CircleLayerTest : BaseStyleTest() {
     assertNotNull("defaultCircleTranslateAnchor should not be null", CircleLayer.defaultCircleTranslateAnchor)
     assertNotNull("defaultCircleTranslateAnchorAsExpression should not be null", CircleLayer.defaultCircleTranslateAnchorAsExpression)
   }
+
+  @Test
+  @UiThreadTest
+  fun getLayerTest() {
+    val filterTestValue = eq {
+      get {
+        literal("undefined")
+      }
+      literal(1.0)
+    }
+    val circleSortKeyTestValue = 1.0
+    val circleBlurTestValue = 1.0
+    val circleColorTestValue = "rgba(0, 0, 0, 1)"
+    val circleOpacityTestValue = 1.0
+    val circlePitchAlignmentTestValue = CirclePitchAlignment.MAP
+    val circlePitchScaleTestValue = CirclePitchScale.MAP
+    val circleRadiusTestValue = 1.0
+    val circleStrokeColorTestValue = "rgba(0, 0, 0, 1)"
+    val circleStrokeOpacityTestValue = 1.0
+    val circleStrokeWidthTestValue = 1.0
+    val circleTranslateTestValue = listOf(0.0, 1.0)
+    val circleTranslateAnchorTestValue = CircleTranslateAnchor.MAP
+
+    val layer = circleLayer("id", "source") {
+      sourceLayer("test")
+      minZoom(10.0)
+      maxZoom(10.0)
```

Comment:
why is this `10.0` not stored in constants as all the other properties?

Reply:
This was copied from other tests, will make these consistent.